articles/industrial-iot/industrial-iot-platform-versions.md

```diff
@@ -5,21 +5,25 @@ author: jehona-m
 ms.author: jemorina
 ms.service: industrial-iot
 ms.topic: overview
-ms.date: 03/08/2021
+ms.date: 03/11/2021
```

Comment:
This is using US date format, so it should be `11/03/2021` instead of `03/11/2021`.